ui/app/js/services/helper_service.js

```diff
@@ -10,6 +10,11 @@ angular.module("app").factory('Helpers', function() {
       }
       return object;
     },
+    selectPageSize: function($scope){
+      if(+$scope.page_size && +$scope.page_size > 0){
```

Comment:
What are there `+` operators before each of the `$scope.page_size` references?

Reply:
Oh, an oversight. I should have removed the `unary` operator. It was only meant to convert the input to integer; but that is only required if I used `text input type`.